app/assets/javascripts/components/revisions/revision_list.jsx

```diff
@@ -62,7 +63,7 @@ const RevisionList = createReactClass({
       references: {
         label: I18n.t('revisions.references'),
         desktop_only: true,
-        info_key: 'metrics.references_doc'
+        info_key: `metrics.references_doc_separated.${ArticleUtils.articlesOrItems(this.props.course.home_wiki.project)}`
```

Comment:
Per my comment on the earlier PR, I think this should be updated so that it continues to use `metrics.references_doc` for articles, and renames the items version something like `references_doc_wikidata`.